### PR TITLE
Finish Mobile Version

### PR DESCRIPTION
**Pull request by Thoth1111**
### Alwyn_Portfolio :Finish Mobile Version
In this part of the project, I built up the middle section of the mobile-first page to include three sections, i.e., work, about myself, and skills section, with the Figma template three as my guide. I did the following to achieve this:

- Set up the grids needed for the sections and set up their alignments and specifications
- Added the introduction title in the first grid and styled it.
- Added the work section details using a combination of grid layout and flexbox and did the styling.
- Added the about myself section using flexbox and styled it
- Added the skills section using flexbox and styled it
- Added the form section using grid templates and styled it.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -37,6 +37,115 @@ <h2>Hey there. I’m Alwyn.<br><span id="green">I’m a software developer</span
         <div id="scroll">
             <input type="image" src="scroll-button.png">
         </div> 
-    </div>   
+    </div>  
+
+    <div class="mid-section">
+        <div class="intro">
+            <h2>My Recent Works</h2>
```
Comment (christianbiring1) [inline Index.html:44]: - [ ] Kindly make sure you stick to the design as much as possible. You may consider using the same typo(font), and color as the one in the original design and for all your elements. not only this specific section

Your design :point_down: 
image

Figma template :point_down: 
image

File: style.css [hunk 2]
```diff
@@ -89,4 +89,209 @@
...
+.button-1 {
+  color: white;
+  height: auto;
+  width: 98px;
+  left: 12px;
+  top: 12px;
+  margin-left: 103px;
+  margin-bottom: 10px;
+  border-radius: 4px;
+  background: #36b37f;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure your button styles look as the ones in Figma. And you may also consider adding the interaction style such as `hover`, `active`, and `disable`
- [ ] Also consider readjusting the color of your elements
`Hint: To get the element style, just click it and on the right you will be able to see the styles applied to it in Figma.` :smile: 
Your design :point_down: 
image

Figma design :point_down: 
image

File: style.css [hunk 3]
```diff
@@ -89,4 +89,209 @@
...
+}
+
+#submit {
+  padding: 12px;
+  text-align: justify;
+}
+
+#emai {
+  font-size: 20px;
+  font-weight: 400;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure you stick to the design. the typo, color, socing(margin, padding) are supposed to be similar to the original design
`Hint: To get a element style, just click it and you will see the style on the right side in Figma`
Your design :point_down: 
image

Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Thoth1111 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are passing :heavy_check_mark: 
- [ ] Descriptive PR `summary` :+1: 
- [ ] Good commit message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Giving your PR a descriptive title. 
Hint: `finish mobile version` would sound more descriptive than `Mid section`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 1 above)
Comment (Thoth1111) [reply Index.html:44]: Hello, I've made the changes. I used Inter font family, the same font used in the template. However, I've corrected the color and sizing mistakes.

**Review by algerina — APPROVED** [review]
Hi @Thoth1111 , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 


 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**